docs/guide/clustering.md

```diff
@@ -35,12 +67,16 @@ For further details on requirements, see [Requirements page](../getting_started/
 
     > Listen on &lt;addr&gt;; default to INADDR\_ANY. &lt;addr&gt; may be specified as host:port. If you don't specify a port number, the value you specified with -p or -U is used. You may specify multiple addresses separated by comma or by using -l multiple times. This is an important option to consider as there is no other way to secure the installation. Binding to an internal or firewalled network interface is suggested.
 
-It is also recommended to use:
+### Shared sessions
 
-- [Solr](solr.md) for better search and performance
-- a CDN for improved performance and faster ping time worldwide
-- active/passive database for failover
-- more recent versions of PHP and MySQL/MariaDB within [what is supported](../getting_started/requirements.md) for your eZ Platform version to get more performance out of each server. Numbers might vary so make sure to test this when upgrading.
+For a [cluster](clustering.md) setup you need to configure sessions to use a back end that is shared between web servers and supports locking.
```

Comment:
`.. , alternatively usage of a load balancer that supports session affinity ("sticky sessions") so a user is served by single server once he gets a session.`

Reply:
https://en.wikipedia.org/wiki/Load_balancing_(computing)#Persistence